core/tg_channel_cache.py

```
@@ -59,37 +93,113 @@ async def _refresh(self, *, force: bool = False) -> None:
                 self._set_unavailable("Telegram client is not authorized.")
                 return
 
+            configured = configured_channel_refs or self._pending_configured_refs or []
             channels_by_ref: dict[str, dict[str, Any]] = {}
+
+            # 1) 先解析已配置频道：快、且对选择器最有用；不依赖全量对话框。
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)